personal.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use OCA\Spreed\Util;
+
+\OC_Util::checkLoggedIn();
+
+$tmpl = new OCP\Template('spreed', 'settings-personal');
+$config = \OC::$server->getConfig();
+$uid = \OC::$server->getUserSession()->getUser()->getUID();
+
+$settings = Util::getTurnSettings($config, $uid);
+//server":"1","username":"2","password":"3","protocols
```

Comment:
Whoops, some debugging leftover - will remove together with other feedback you might have.